[2022-09-26T11:23:01.923+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2021-10-10T00:00:00+00:00 [queued]>
[2022-09-26T11:23:01.955+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2021-10-10T00:00:00+00:00 [queued]>
[2022-09-26T11:23:01.956+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T11:23:01.956+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2022-09-26T11:23:01.957+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-09-26T11:23:01.999+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): model.migrate_to_dbt.view_data> on 2021-10-10 00:00:00+00:00
[2022-09-26T11:23:02.007+0000] {standard_task_runner.py:54} INFO - Started process 2106 to run task
[2022-09-26T11:23:02.014+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'model.migrate_to_dbt.view_data', 'scheduled__2021-10-10T00:00:00+00:00', '--job-id', '1216', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpf8rtzacv']
[2022-09-26T11:23:02.014+0000] {standard_task_runner.py:83} INFO - Job 1216: Subtask model.migrate_to_dbt.view_data
[2022-09-26T11:23:02.016+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dbt_dag.py
[2022-09-26T11:23:02.127+0000] {task_command.py:384} INFO - Running <TaskInstance: dbt_dag.model.migrate_to_dbt.view_data scheduled__2021-10-10T00:00:00+00:00 [running]> on host 3c654011b3f0
[2022-09-26T11:23:02.308+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=isaaclucky88@gmail.com
AIRFLOW_CTX_DAG_OWNER=astronomer
AIRFLOW_CTX_DAG_ID=dbt_dag
AIRFLOW_CTX_TASK_ID=model.migrate_to_dbt.view_data
AIRFLOW_CTX_EXECUTION_DATE=2021-10-10T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-10T00:00:00+00:00
[2022-09-26T11:23:02.309+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-09-26T11:23:02.310+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n            cd /opt/***/migrate_to_dbt &&\n            dbt --no-write-json run --target prod --models view_data\n            ']
[2022-09-26T11:23:02.328+0000] {subprocess.py:86} INFO - Output:
[2022-09-26T11:23:02.332+0000] {subprocess.py:93} INFO - /bin/bash: line 3: dbt: command not found
[2022-09-26T11:23:02.333+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-09-26T11:23:02.354+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-09-26T11:23:02.361+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=dbt_dag, task_id=model.migrate_to_dbt.view_data, execution_date=20211010T000000, start_date=20220926T112301, end_date=20220926T112302
[2022-09-26T11:23:02.385+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 1216 for task model.migrate_to_dbt.view_data (Bash command failed. The command returned a non-zero exit code 127.; 2106)
[2022-09-26T11:23:02.425+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-09-26T11:23:02.490+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
